app/src/main/java/org/simple/clinic/summary/bloodpressures/newbpsummary/NewBloodPressureSummaryViewModel.kt

```diff
@@ -5,12 +5,16 @@ import java.util.UUID
 
 data class NewBloodPressureSummaryViewModel(
     val patientUuid: UUID,
-    val bloodPressures: List<BloodPressureMeasurement>?
+    val bloodPressures: List<BloodPressureMeasurement>?,
+    val bloodPressuresCount: Int
```

Comment:
It is not clear from the names of these two properties that they are actually different. Someone might look at this model in the future and wonder why we are tracking the count of BPs separately when we already have the list of BPs available.

I think we can name these properties better in order to expose these details. What do you think of:
- `bloodPressures` -> `latestBloodPressuresToDisplay`
- `bloodPressuresCount` -> `totalRecordedBloodPressureCount`